src/test/ui/impl-trait/issue-86465.stderr

```diff
@@ -1,14 +1,14 @@
 error: concrete type differs from previous defining opaque type use
-  --> $DIR/issue-86465.rs:5:1
+  --> $DIR/issue-86465.rs:6:5
    |
-LL | fn f<'t, 'u>(a: &'t u32, b: &'u u32) -> (X<'t, 'u>, X<'u, 't>) {
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `&'a u32`, got `&'b u32`
+LL |     (a, a)
+   |     ^^^^^^ expected `&'a u32`, got `&'b u32`
    |
 note: previous use here
-  --> $DIR/issue-86465.rs:5:1
+  --> $DIR/issue-86465.rs:6:5
    |
-LL | fn f<'t, 'u>(a: &'t u32, b: &'u u32) -> (X<'t, 'u>, X<'u, 't>) {
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+LL |     (a, a)
+   |     ^^^^^^
```

Comment:
Shame that we don't provide more info here to help people understand what's going on.

Reply:
Maybe we could special case if prev and the primary spans are the same? 🤷‍♀️

Reply:
I hope I can figure out how to point to the individual tuple fields at some point, but yea, maybe special casing for equal spans is the best for now